core/tests/protractor/i18n.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests of the i18n platform and the completion of
+ * translations.
+ *
+ * @author Milagro Teruel (milagro.teruel@gmail.com)
+ */
+
+var general = require('../protractor_utils/general.js');
+var gallery = require('../protractor_utils/gallery.js');
+var users = require('../protractor_utils/users.js');
+var workflow = require('../protractor_utils/workflow.js');
+
+var checkForTranslationIds = function (selector) {
+  element(by.css('.oppia-base-container')).then(function (value) {
+    value.getInnerHtml().then(function (promiseValue) {
+      expect(promiseValue).not.toContain('I18N_GALLERY');
```

Comment:
Yes! I added one positive check. Also, the gallery tests have some of those too.